index.html

```diff
@@ -14,6 +15,19 @@ <h1 id="logo">Aziz Mejri</h1>
       <button type="button" class="menu-bars">
         <i class="fas fa-bars"></i>
       </button>
+      <nav id="mobile-menu" class="not-active">
+        <div>
+          <i id="close" class="fas fa-xmark"></i>
+          <ul>
+            <li><a class="mobile-link" href="#about-me">Portoflio</a></li>
+            <li><a class="mobile-link" href="##contacts">About</a></li>
+            <li>
+              <a class="mobile-link" href="#contact">Contact</a>
+            </li>
+            <i id="close" class="fas fa-xmark"></i>
```

Comment:
- [ ] This line of code is a repetition and it has caused a defect in the overall design of your mobile menu. i will attach below the figma design and your current design so you can see the difference and what is expected of you.

this is the figma design having an `X` to close the menu
![figma-des](https://user-images.githubusercontent.com/89055502/233188948-f0803801-6fb7-4cc5-85dc-a80fafcac23e.png)

This is your design and it has two `X`, one above the portfolio and one below the contact which it isn't supposed to have.
![faith-des](https://user-images.githubusercontent.com/89055502/233189220-1642fc0e-cbaa-488a-8859-824d61d3a5e9.png)
- [ ] i kindly suggest that you correct this error by removing the `X` below the contact. you can do this by removing line 27 of your `index.html` file so your project matches the figma design and looks professional. Recruiters want to know you can produce a given figma design and having errors like this won't be of great help to you. :+1: